smslibrary/src/test/java/com/eis/smslibrary/SMSManagerTest.java

```diff
@@ -17,8 +18,8 @@
 import static org.junit.Assert.assertEquals;
 import static org.mockito.Matchers.anyString;
 import static org.mockito.Matchers.isNull;
+import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
-import static org.powermock.api.mockito.PowerMockito.mock;
```

Comment:
what's the reason for this change?

Reply:
Mock objects should come from Mockito instead of PowerMockito, that should be used only for static functions.

Reply:
`org.powermock.api.mockito.PowerMockito.mock()` mocks classes just like Mockito's `mock()`, not static methods. `org.powermock.api.mockito.PowerMockito.mockStatic()` is the method used for mocking a class' static methods.
Anyway, I just read the documentation for both `mock()` methods and PowerMockito's version supports mocking of final and native methods, which I guess is not possible with Mockito since this is not mentioned there. So I would keep Mockito's version because it's more restrictive, you can mark this as resolved.